### PR TITLE
Added most Betrayal uniques

### PR DESCRIPTION
**Pull request by Patchumz**
Added Bitterbind Point, Cinderswallow, Flamesight, Galesight, Hyperboreus, Solstice Vigil, Crimson Storm, Devouring Diadem, Thundersight, Vivinsect, Whakatutuki o Matua, and Worldcarver.
(Including veiled mod variations)

Updated Matua Tupuna's Source description

### CODE REVIEW

**Comment by Patchumz** [comment]
> 
> 
> Just trying to double check everything, but having a hard time finding a source with every variant available to Veiled mods on the new unique items. If you don't mind, do you know what source you were pulling that information from?

Yeah, I was going through trade mostly. Since we're deep into the league, there's plenty of every variant. The only one I excluded was +1 totem on Bitterbind because it's getting removed in a week anyways.

I don't know of any source that lists all the possible veiled mods for each individual unique. I wish I did though; it would speed up the process a lot.

File: Data/Uniques/belt.lua [hunk 1]
```diff
@@ -224,6 +224,35 @@ Requires Level 40
...
+Variant: Trap Throwing Speed
+Variant: Energy Shield Regen
+Variant: Lucky Crit Chance while Focussed
+Requires Level 60
++(25–40) to maximum Life
++(30–40)% to Cold Resistance
+Chill nearby Enemies when you Focus, causing 30% reduced Action Speed
+Focus has (15–25)% increased Cooldown Recovery Speed
+(50–70)% increased Damage with Hits and Ailments against Chilled Enemies
+{variant:1}{crafted}(1-2)% of Life Regenerated per second during any Flask Effect
```
Comment (PJacek) [inline Data/Uniques/belt.lua]:
```suggestion
{variant:1}{crafted}(1.0-2.0)% of Life Regenerated per second during any Flask Effect
```
Allows slider to work correctly for fractions. This mod can have the value of 1.5

File: Data/Uniques/bow.lua [hunk 2]
```diff
@@ -38,6 +38,44 @@ Adds 25 to 50 Fire Damage
...
+50% chance to inflict Bleeding on Critical Strike with Attacks
+Enemies you inflict Bleeding on grant (60–100)% increased Flask Charges
+Adds (100–120) to (150–165) Physical Damage against Bleeding Enemies
+50% chance to Maim Enemies on Critical Strike with Attacks
+{variant:1}{crafted}+(18-45)% Critical Strike Multiplier while there is a Rare or Unique Enemy Nearby
+{variant:2}{crafted}(11-22)% increased Attack Speed while a Rare or Unique Enemy is Nearby
+{variant:3}{crafted}(5-6)% increased Damage per Frenzy Charge
+{variant:4}{crafted}(5-6)% increased Damage per Power Charge
+{variant:5}{crafted}(5-6)% increased Damage per Endurance Charge
+{variant:6}{crafted}+(51-250) to Accuracy Rating
```
Comment (PJacek) [inline Data/Uniques/bow.lua]:
```suggestion
{variant:6}{crafted}+(30-250) to Accuracy Rating
```
Can have lower minimal value

File: Data/Uniques/bow.lua [hunk 3]
```diff
@@ -38,6 +38,44 @@ Adds 25 to 50 Fire Damage
...
+{variant:6}{crafted}+(10-18)% to Quality
```
Comment (PJacek) [inline Data/Uniques/bow.lua]:
```suggestion
{variant:6}{crafted}+(7-18)% to Quality
```
Can have lower minimal value (same as Attack Speed/Quality variant)

File: Data/Uniques/bow.lua [hunk 4]
```diff
@@ -38,6 +38,44 @@ Adds 25 to 50 Fire Damage
...
+{variant:7}{crafted}(8-16)% increased Attack Speed
+{variant:7}{crafted}+(7-18)% to Quality
+{variant:8}{crafted}(8-16)% increased Attack Speed
+{variant:8}{crafted}10% chance to Trigger Level 1 Blood Rage when you Kill an Enemy
+{variant:9}{crafted}Minions have (16-21)% increased Attack Speed
```
Comment (PJacek) [inline Data/Uniques/bow.lua]:
```suggestion
{variant:9}{crafted}Minions have (16-28)% increased Attack Speed
```
Can have higher maximum value

File: Data/Uniques/bow.lua [hunk 5]
```diff
@@ -38,6 +38,44 @@ Adds 25 to 50 Fire Damage
...
+{variant:9}{crafted}Minions have (16-21)% increased Cast Speed
```
Comment (PJacek) [inline Data/Uniques/bow.lua]:
```suggestion
{variant:9}{crafted}Minions have (16-28)% increased Cast Speed
```
Can have higher maximum value

File: Data/Uniques/bow.lua [hunk 6]
```diff
@@ -38,6 +38,44 @@ Adds 25 to 50 Fire Damage
...
+{variant:10}{crafted}(4-12)% chance to deal Double Damage
+{variant:11}{crafted}(13-36)% chance to deal Double Damage while Focussed
+{variant:12}{crafted}Trigger a Socketed Spell when you Use a Skill
+]],[[
```
Comment (PJacek) [inline Data/Uniques/bow.lua]: Missing variant with Cast Speed/Arcane Surge on Kill
```
(9-13)% increased Cast Speed
10% chance to gain Arcane Surge when you Kill an Enemy
```

File: Data/Uniques/gloves.lua [hunk 7]
```diff
@@ -383,6 +383,16 @@ Requires Level 27, 22 Str, 22 Dex
...
 You gain Onslaught for 5 seconds on using a Vaal Skill
 ]],[[
+Worldcarver
+Dragonscale Gauntlets
+Requires Level 67, 51 Str, 51 Dex
+Trigger Level 20 Arcane Wake after Spending a total of 200 Mana
++(200–300) to Accuracy Rating
+(120–150)% increased Armour and Evasion
++(60–80) to maximum Life
+3% increased Attack Speed for each Map Item Modifier affecting the Area
```
Comment (PJacek) [inline Data/Uniques/gloves.lua]: According to pathofexile.com/trade Attack Speed should be after Attack Damage. Wiki probably has it wrong.

File: Data/Uniques/bow.lua (hunk 6 above)
Comment (Patchumz) [reply Data/Uniques/bow.lua]: Added.

File: Data/Uniques/gloves.lua (hunk 7 above)
Comment (Patchumz) [reply Data/Uniques/gloves.lua]: Yup. Trade has speed after damage. Poedb had it the other way around. I'll change it anyways.

File: Data/Uniques/ring.lua [hunk 8]
```diff
@@ -785,19 +785,51 @@ Requires Level 65
...
+{variant:3}{crafted}+(8-15)% to Fire and Chaos Resistances
+{variant:4}{crafted}+(6-17) to Strength and Intelligence
+{variant:5}{crafted}+(6-17) to Dexterity and Intelligence
+{variant:6}{crafted}+(6-17) to Strength and Dexterity
+{variant:7}{crafted}(11-30)% increased Effect of non-Damaging Ailments on Enemies
+{variant:8}{crafted}Shock nearby Enemies for (2-4) Seconds when you Focus
+{variant:9}{crafted}+1 to Minimum Frenzy Charges
+{variant:10}{crafted}+1 to Minimum Power Charges
+{variant:11}{crafted}+1 to Minimum Endurance Charges
+]],[[
```
Comment (PJacek) [inline Data/Uniques/ring.lua]: Missing variant with Attack and Cast Speed
```
(3-6)% increased Attack and Cast Speed
```

Comment (Patchumz) [reply Data/Uniques/ring.lua]: Added.

File: Data/Uniques/shield.lua [hunk 9]
```diff
@@ -375,6 +375,33 @@ You take Chaos Damage instead of Physical Damage from Bleeding
...
+{variant:1}{crafted}Adds (3-12) to (5-16) Cold Damage
+{variant:1}{crafted}Adds 1 to (7-24) Lightning Damage
+{variant:2}{crafted}Adds (3-12) to (5-16) Fire Damage
+{variant:2}{crafted}Adds 1 to (7-24) Lightning Damage
+{variant:3}{crafted}Adds (3-12) to (5-16) Fire Damage
+{variant:3}{crafted}Adds (3-12) to (5-16) Cold Damage
+{variant:4}{crafted}(20-40)% increased Energy Shield
+{variant:4}{crafted}+(10-28) to maximum Life
+{variant:5}{crafted}+(500-3000) to Armour during Soul Gain Prevention
+{variant:6}{crafted}+1 to Level of Socketed Support Gems
```
Comment (PJacek) [inline Data/Uniques/shield.lua]:
```suggestion
{variant:6}{crafted}+(1-2) to Level of Socketed Support Gems
```
Can have higher maximum value

**Comment by Patchumz** [comment]
I avoided Paradoxica and Queen's Hunger specifically for the problem cases you bring up. I thought about just added the raw items without veiled mods, but figured that would be a bit confusing and inconsistent. 

Instead I'm assuming Openarl has an idea on what he wants to do with 2+ variants (and prefixes/suffixes) now that there's two different cases of it in the game, with potential for more. I decided to not make a messy hackjob unless I really had to.

File: Data/Uniques/flask.lua [hunk 10]
```diff
@@ -107,6 +107,31 @@ Chaos Damage does not bypass Energy Shield during effect
 Removes all but one Life on use
 Removed life is regenerated as Energy Shield over 2 seconds
 ]],[[
+Cinderswallow Urn
+Silver Flask
+League: Betryal
```
Comment (PJacek) [inline Data/Uniques/flask.lua]:
```suggestion
League: Betrayal
```
A typo

**Comment by Openarl** [comment]
Thanks! This helps a lot.

**Pull request merged**